scout/commands/serve.py

```diff
@@ -33,6 +32,9 @@ def serve(host, port, debug, livereload, test):
         sys.exit("No database connection available")
 
     if livereload:
+        LOG.warning(
```

Comment:
What about marking the whole serve command as deprecated instead? Like [here](https://github.com/Clinical-Genomics/scout/blob/dd706fba88d0815cb05ad4eea9e93520baf7f2ca/scout/commands/load/report.py#L14)

Reply:
Hm, I have a feeling the `serve` command is "nice" for new users/trying stuff out. Perhaps we could even just change it to fire up the app using gunicorn implicitly. But maybe it's just my habit speaking. Let's deprecate it. That way we are free to remove it, but can always change our minds and fix it instead if it seems possible.